### ISSUE TITLE
Add the support to the find_package for rFSM library

### ISSUE THREAD

**Issue by GiulioRomualdi** [robotology/rfsmTools#14]
It would be nice to have the possibility to link the rFSM library even if is it not installed inside the system path (i.e. `/usr/bin/ld`). 

Indeed if I try to compile the [example](https://github.com/robotology/rfsmTools/tree/master/examples) I retrieved the following error
```
/usr/bin/ld: cannot find -lrFSM
collect2: error: ld returned 1 exit status
CMakeFiles/rfsmTest.dir/build.make:94: recipe for target 'rfsmTest' failed
make[2]: *** [rfsmTest] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/rfsmTest.dir/all' failed
make[1]: *** [CMakeFiles/rfsmTest.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```
I think that this error can be easily solved by adding the `rFSMConfig.cmake.in` file and changing a bit the `CMakeLists.txt`.

If you are interested in this feature I can propose a PR. Let me known.

**Comment by traversaro** [comment]
You are probably  aware of this @GiulioRomualdi , but https://github.com/robotology/how-to-export-cpp-library is always updated with the latest trends in how to export C++ packages using CMake, if you need a reference.

**Comment by Nicogene** [comment]
Closed by #15
